Libs/MRML/Core/vtkMRMLScriptedModuleNode.cxx

```diff
@@ -109,7 +109,11 @@ void vtkMRMLScriptedModuleNode::CopyContent(vtkMRMLNode* anode, bool deepCopy/*=
     }
 
   this->SetModuleName(node->GetModuleName());
-  this->Parameters = node->Parameters;
+  if (this->Parameters != node->Parameters)
+  {
```

Comment:
Indentation does not follow the current coding conventions (we will switch to a more common indentation style all at once, but we keep the current one for now, for consistency)

Otherwise it looks good to me, too.

Reply:
Fixed indentation.